### ISSUE TITLE
PHP 7.4 compatibility issue

### ISSUE THREAD

**Issue by Nuranto** [spatie/schema-org#109]
I'm currently migrating to PHP7.4  (GA release will be on November, 28th) on my dev environment.
I get an error : 
`Function ReflectionType::__toString() is deprecated` thrown on a ` $graph->organization()` call

I think I've found a fixe, but not sure about this enough to make a PR : 
```
$this->getOrCreate($type);
```
=>
```
$this->getOrCreate($type ? $type->getName() : null);
```

(in `Spatie\SchemaOrg\Graph`)

**Comment by Gummibeer** [comment]
Thanks for pointing out. Right now PHP7.4 isn't supported but after reading the docs the `__toString()` method is deprecated since PHP5.5 😮 
So will try to find a fix. The problem is that `getName()` is only available on `ReflectionNamedType`.

**Comment by Nuranto** [comment]
In english version of the doc, it says PHP7.1.0, not 5.5... Probably a bug in DE doc ^^

Anyway, thanks for the fix to come !

**Comment by Nuranto** [comment]
Similar issue on symfony repo : https://github.com/symfony/symfony/issues/32397
They used the `getName()` solution, even if they were also dealing with `ReflectionType` class.. I don't get it either, but I tried the fix I proposed earlier, and it fixes the issue.

Hope it helps.
Note : I tested this only on a PHP7.4 env.

EDIT :  If I `dd(get_class($type));` it seems that get `getReturnType` returns a `ReflectionNamedType`, and not a `ReflectionType` as the doc says. (Probably the same for `ReflectionParameter::getType()` by the way)
